modules/google_search_20/Readme.md

```diff
@@ -0,0 +1,5 @@
+Google Custom Search Module 2.0.
+
+https://developers.google.com/custom-search/docs/element
+The current solution is based on
+Results only <gcse:searchresults-only>
```

Comment:
Please add more information
1) where to put the API key (i.e in the associated json file).
2) how to get such API key ... put a link to the related google site.